tests/e2e/specs/get-started.test.js

```diff
@@ -0,0 +1,54 @@
+const { merchant } = require( '@woocommerce/e2e-utils' );
+
+describe( 'Merchant who is getting started', () => {
+	beforeAll( async () => {
+		await merchant.login();
+	} );
+
+	it( 'Clicks on the Marketing > GLA link, clicks on the call-to-action setup button to go to the Setup MC page', async () => {
+		// hover at the marketing link to open popup submenu.
+		const marketingLink = (
+			await page.$x( "//a[contains(., 'Marketing')]" )
+		 )[ 0 ];
+		await marketingLink.hover();
+
+		// the submenu is in the DOM but not visible with the use of absolute positioning.
+		// we wait for submenu to open by checking for "opensub" CSS class in Marketing menu item.
+		await page.waitForSelector(
+			'li#toplevel_page_woocommerce-marketing.opensub'
+		);
+
+		// the submenu is now opened, the GLA sub menu item is now visible to the user,
+		// we can call `click` now.
+		const glaLink = (
+			await page.$x( "//a[text()='Google Listings & Ads']" )
+		 )[ 0 ];
+		await glaLink.click();
+
+		await page.waitForNavigation();
+		const pageTitle = await page.title();
+		expect( pageTitle ).toContain( 'Google Listings & Ads' );
+
+		// click on the call-to-action button.
+		const setupButton = (
+			await page.$x( "//a[text()='Set up free listings in Google']" )
+		 )[ 0 ];
+		await setupButton.click();
+		await page.waitForNavigation();
+
+		// check we are in the Setup MC Step 1 page.
+		const topBarText = (
+			await page.$x(
+				"//*[text()='Get started with Google Listings & Ads']"
+			)
+		 )[ 0 ];
+		expect( topBarText ).toBeTruthy();
+
+		// there are some API calls running in the page before the steps are displayed.
+		// wait for the text in Step 1 to show up.
+		const step1Text = await page.waitForXPath(
+			"//*[text()='Set up your accounts']"
+		);
+		expect( step1Text ).toBeTruthy();
```

Comment:
:scroll: :nail_care: 
WDYT of adding something like [`jest-puppeteer`](https://github.com/smooth-code/jest-puppeteer) for more human-friendly assertions?

```js
await expect(page).toClick('button', { text: 'Set up free listings in Google' })

await expect(page).toMatch('Set up your accounts')
```

(I'm not pushing this particular library, I've never used this, I'm just looking for something to make Jest assertions more usable)

Reply:
Actually I had it in the [early beginning](https://github.com/woocommerce/google-listings-and-ads/pull/801/commits/8101a44fccd0b0d1f34592dff3959e3b02f3d650#diff-2f32f68635ef57d340aacf8d133720c598daa8ef954f9dd69f1cd19577479f65R27-R29) like this:

```js
		await expect( page.title() ).resolves.toMatch(
			'Google Listings & Ads'
		);
```

Along the way I changed it because I wasn't familiar with it. Now that you brought it up, I just look deeper into [`jest-puppeteer`](https://github.com/smooth-code/jest-puppeteer) and its [`expect-puppeteer`](https://github.com/smooth-code/jest-puppeteer/blob/master/packages/expect-puppeteer/README.md), and it seems to make sense now. I can try with that API in the next iteration, sounds good for you? 🤔

Reply:
And speaking of better assertion and API, I don't quite like `waitForXPath`, and thought about having something similar to [Testing Library's `getByText`](https://testing-library.com/docs/queries/about/). I looked into its source code briefly and thought about maybe having that in the next iteration too. `expect-puppeteer` would probably be enough to get the job done.

Reply:
I like the idea of adding it in a separate PR.

>  I don't quite like `waitForXPath`, ...

I'd stick to `expect-puppeteer` as long as it's sufficient, as we already have it as an (indirect) depenedncy.